frontend/components/auction/debt/DebtAuctionTransactionFlow.vue

```diff
@@ -36,7 +36,7 @@
                 :desired-amount="auction.bidAmountDai"
                 :allowance-amount="allowanceDai"
                 :network="network"
-                :token-address="tokenAddress"
+                :token-address="tokenAddress || ''"
```

Comment:
Please instead not require `tokenAddress` prop and properly handle the text that displays token address in case it is empty (eg do not display this part of the text, display it as loading, etc)